engine/core.py

```diff
@@ -126,6 +128,13 @@ def create_sprite(
         _spec = self._spec.sprites[spec_name]
         self.world.create_sprite(_spec, name, start_location, script)
 
+    def get_sprites(self, name: str) -> Iterable[Sprite]:
+        """Gets all sprites with the given name."""
+        if self.world is None:
+            raise GameNotInitializedError()
+
+        return self._spec.sprites.get(name, None)
```

Comment:
Got it. Is the `get_sprites` supposed to return the list of sprites or a dictionary of sprites that match?

Reply:
It will return an _iterable_ of sprites. A list is one type of iterable, but there are others (sets, dicts, tuples, generators). By saying "iterable" instead of "list" it allows us to be more flexible in the types we'll can use.